packages/console/src/components/AppContent/index.tsx

```diff
@@ -51,10 +55,10 @@ const AppContent = () => {
 
   return (
     <div className={styles.app}>
-      <Topbar />
+      <Topbar className={conditional(scrollTop > 0 && styles.topbarShadow)} />
```

Comment:
nit: We usually use `classNames` for className. So:
```tsx
<Topbar className={classNames(scrollTop && styles.topbarShadow)} />
```

Reply:
<img width="1048" alt="image" src="https://user-images.githubusercontent.com/10806653/176831279-1ac82d35-de7c-41e2-850f-86cdd65ee3cd.png">

Reply:
updated.